internal/metrics/metrics.go

```diff
@@ -14,6 +14,14 @@ const (
 	ValueSucceededTrue = "true"
 	// ValueSucceededFalse is value False for metric label failed.
 	ValueSucceededFalse = "false"
+	// ValueSucceededPreempted is value Preempted for metric label preempted.
+	ValueSucceededPreempted = "preempted"
```

Comment:
These label values make no sense. How can you have succeeded with a failure reason?